.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -28,6 +28,6 @@ body:
   - type: textarea
     attributes:
       label: Additional context
-      description: Add any other context about the problem here.
```

Comment:
i think we should just add this as a separate field, including OS version, OS, modrinth app version, etc

Reply:
Still think this is an issue, it should ask in separate fields/have it's own dedicated one.